molecule/scenario_with_local_binary_directory/tests/test_default.py

```diff
@@ -0,0 +1,10 @@
+"""Role testing files using testinfra."""
+
+
+def test_hosts_file(host):
+    """Validate /etc/hosts file."""
+    f = host.file("/etc/hosts")
+
+    assert f.exists
+    assert f.user == "root"
+    assert f.group == "root"
```

Comment:
I believe this is a leftover file - can you please remove it from the PR

Reply:
Yes i will remove it